gae/room/staffposition_api.py

```diff
@@ -0,0 +1,76 @@
+"""API for StaffPositions and their (hourly,mileage) info."""
+
+from google.appengine.ext import ndb
+
+from protorpc import message_types
+from protorpc import messages
+from protorpc import remote
+from protorpc.wsgi import service
+
+import base_api
+import ndb_models
+import protorpc_messages
+
+package = 'rooms'
+
+
+class RateAfterDate(messages.Message):
+  date = messages.StringField(1)
+  rate = messages.FloatField(2)
+
+
+class StaffPosition(messages.Message):
+  id = messages.IntegerField(1)
+  position_name = messages.StringField(2)
+  hourly_rates = messages.MessageField(RateAfterDate, 3, repeated=True)
+  mileage_rates = messages.MessageField(RateAfterDate, 4, repeated=True)
+
+
+class StaffpositionApi(base_api.BaseApi):
+
+  def _staffposition_put(self, request, staffposition=None):
```

Comment:
Most of this method is about copying stuff from request into model. My convention for this stuff is to put it in a "messagetomodel" method like you see in custom_api and protorpc_messages modules

Reply:
Can this method stay right here within StaffpositionApi but be named:

```
def message_to_model(self, msg, mdl):
    ...
```

Reply:
Yeah something like that.  The name should be consistent with conventions in protorpc_messages which are like OrderModelToMessage()
And then the .put() call moves outside this method.